tests/Type/Symfony/data/messenger_handle_trait.php

```diff
@@ -111,3 +111,32 @@ public function __invoke()
         assertType('mixed', $this->handle(new MultiHandlersForTheSameMessageQuery()));
     }
 }
+
+class QueryBus {
+    use HandleTrait;
+
+    public function dispatch(object $query): mixed
+    {
+        return $this->handle($query);
+    }
```

Comment:
We could also add this test case with changed function name.

```suggestion
    use HandleTrait {
        handle as private handleQuery;
    }

    public function handle(object $query): mixed
    {
        return $this->handleQuery($query);
    }
```